storage/remote/queue_manager.go

```diff
@@ -97,12 +97,12 @@ var (
 		},
 		[]string{queue},
 	)
-	queueCapacity = prometheus.NewGaugeVec(
+	shardCapacity = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Namespace: namespace,
 			Subsystem: subsystem,
-			Name:      "queue_capacity",
-			Help:      "The capacity of the queue of samples to be sent to the remote storage.",
+			Name:      "shard_capacity",
+			Help:      "The capacity of each shard of the queue used for parallel sending to the remote storage.",
```

Comment:
We can always just ditch this change from this PR, its not important to the overall changes.